src/FormsSubmit.vue

```diff
@@ -46,8 +46,45 @@ export default {
 		return {
 			form: loadState('forms', 'form'),
 			isLoggedIn: loadState('forms', 'isLoggedIn'),
+			isEmbedded: loadState('forms', 'isEmbedded', false),
 			shareHash: loadState('forms', 'shareHash'),
 		}
 	},
+
+	mounted() {
+		if (this.isEmbedded) {
```

Comment:
```suggestion
		if (this.isEmbedded) {
			subscribe('forms:last-updated:set', (id) => {
				window.parent?.postMessage({
					type: 'form-saved',
					payload: {
						id,
					},
				}, '*')
			})
```
To be possible trigger the saved event at window parent